lib/ValgrindBenchmarkTools/src/ValgrindBenchmarkTools.jl

```diff
@@ -0,0 +1,30 @@
+module ValgrindBenchmarkTools
+
+using Valgrind, Reexport
+@reexport using BenchmarkTools
+
+@inline function prehook()
+    Valgrind.Callgrind.zero_stats()
+    Valgrind.Callgrind.start_instrumentation()
+end
+
+@inline function posthook(id)
+    Valgrind.Callgrind.stop_instrumentation()
+    Valgrind.Callgrind.dump_stats_at(id)
```

Comment:
I'm guessing you want this in the `sample_result` hook instead if you're trying to store the result of `dump_stats_at` as part of your benchmark results.